src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -427,4 +426,18 @@ describe('TimeEntriesComponent', () => {
       expect(entryData.customer_name).toContain('ioet Inc.');
     });
   }));
+
+  it('Should the entry be null if the flag is true', () => {
```

Comment:
`should clear the entry If I add a new entry after editing an existing time-entry.`